SequenceOf+Linq.swift

```diff
@@ -103,29 +103,28 @@ extension SequenceOf {
                     }
                 }
                 return generator.next()
-                })
             })
+        })
     }
     
     func SkipWhile(f: (T) -> Bool) -> SequenceOf<T> {
         return SequenceOf<T>({() -> GeneratorOf<T> in
             var generator = self.generate()
-            var conditionMet = true
+            var conditionMet = false
```

Comment:
This code is logically correct but, stylistically, the semantics of "conditionMet" don't make sense when it starts out false. I'll do a quick change to swap true/false and remove the '!' from the while loop then merge.